python_modules/dagster/dagster/_core/definitions/freshness_checks/last_update.py

```diff
@@ -22,8 +22,8 @@ def build_last_update_freshness_checks(
     deadline_cron: Optional[str] = None,
     timezone: str = DEFAULT_FRESHNESS_TIMEZONE,
     severity: AssetCheckSeverity = DEFAULT_FRESHNESS_SEVERITY,
-) -> Sequence[AssetChecksDefinition]:
-    r"""For each provided asset, constructs a freshness check definition.
+) -> AssetChecksDefinition:
+    r"""Constructs an `AssetChecksDefinition` that checks the freshness of the provided assets.
```

Comment:
note AssetChecksDefinition may go away- it wraps AssetsDefinition

Reply:
will change description